internals/core_enums.py

```diff
@@ -150,6 +150,19 @@
     FEATURE_TYPE_ENTERPRISE_ID: 'New Feature or removal affecting enterprises',
 }
 
+# When this Web Platform feature ships, how will it be rolled out to users?
+ROLLOUT_100  = 0
+ROLLOUT_0_THEN_100 = 1
+ROLLOUT_GRADUAL = 2
+ROLLOUT_OTHER = 3
+
+ROLLOUT_PLAN_DISPLAYNAMES = collections.OrderedDict([
+  (ROLLOUT_100, 'Will ship enabled for all users'),
+  (ROLLOUT_0_THEN_100, '(RARE) Ships disabled, then flips on for all users'),
+  (ROLLOUT_GRADUAL, '(RARE) Experiment users ramp up over time'),
+  (ROLLOUT_OTHER, "(RARE) It's complicated or unusual"),
+])
```

Comment:
Why is an OrderedDict used here? I don't see anything that references the order here

Reply:
Good point.  I was just copying the existing pattern.  However, in python these days, all dicts are ordered.  I'll try changing it.